Doc/whatsnew/3.12.rst

```diff
@@ -413,6 +413,10 @@ Deprecated
   is no current event loop set and it decides to create one.
   (Contributed by Serhiy Storchaka and Guido van Rossum in :gh:`100160`.)
 
+* In accordance with PEP 699, the ``ma_version_tag`` field in :c:type:`PyDictObject`
```

Comment:
Woops, I also just realised we can markup the PEP to link to it. Sorry, could you adjust the news too?
```suggestion
* In accordance with :pep:`699`, the ``ma_version_tag`` field in :c:type:`PyDictObject`
```